resources/js/tasks/components/TasksList.vue

```diff
@@ -376,7 +376,8 @@ export default {
                   "&per_page=" +
                   this.perPage +
                   filterParams +
-                  this.getSortParam(),
+                  this.getSortParam() +
+                  "&non_system=true",
```

Comment:
@eiresendez I think non-system should be the default. I think you can remove this and `->when(...` above and just use $query->nonSystem() by itself

Reply:
@nolanpro I found that there were other places where that API endpoint was called and they needed to find the system tasks. For this reason, I set the scope only for /tasks (web)